Udpate README (#253)
openwsn-berkeley/lakers: README.md

- lightweight: full authenticated key exchange in as few as 101 bytes
- secure: mutual authentication, forward secrecy, and identity protection
- transport-agnostic: can be used on scenarios with or without IP connectivity; a common way to transport EDHOC is [over reliable CoAP](https://www.rfc-editor.org/rfc/rfc9528.html#coap)
- code re-use: achieved due to re-using technologies common in IoT scenarios, such as COSE, CBOR, and CoAP
- finally, a main use case of EDHOC is to establish an [OSCORE](https://datatracker.ietf.org/doc/rfc8613/) security context

# lakers features

- easy to use, typestated API for Initiator and Responder
- microcontroller-optimized: `no_std`, no heap allocations, zero-dependencies (other than crypto backends)
- configurable crypto backends
- bindings for [C](https://github.com/openwsn-berkeley/lakers/releases/) and [Python](https://pypi.org/project/lakers-python/)
- support for EDHOC extensions, including [zero-touch authorization](https://datatracker.ietf.org/doc/draft-ietf-lake-authz/)

It currently supports authentication mode STAT-STAT and Cipher Suite 2 (AES-CCM-16-64-128, SHA-256, 8, P-256, ES256, AES-CCM-16-64-128, SHA-256).

# Getting started

> To use `lakers` in your Rust project, add it to your Cargo.toml: `lakers = "0.5.1"` (check for the [latest version here](https://crates.io/crates/lakers)).

Reviewer: That doesn't age well.
```suggestion
To use `lakers` in your Rust project, add it to your Cargo.toml using `cargo add lakers`.
```
